lib/importer/nunjucks/importer/macros/row_selector.njk

```diff
@@ -0,0 +1,27 @@
+{% from "importer/macros/table_view.njk" import importerTableView %}
```

Comment:
This macro doesn't appear to be used in this file... should it be?

Reply:
No, but it was in an earlier iteration. Will fix.

Reply:
Removed in c865f77